tutorials/kafka-live-viewer-profiles/introduction.md

```diff
@@ -0,0 +1,56 @@
+---
+position: 1
+title: Introduction
+---
+
+# Snowplow solution accelerator: live viewer profiles
+
+## Introduction
+
+Welcome to the **live viewer profiles** solution accelerator for video streaming! 
+
+This accelerator demonstrates how to build a real-time use case leveraging **Snowplow event data** to create live viewer profiles for a video streaming site. By combining Snowplow's streaming pipeline with **Apache Kafka**, a **Java application** and **AWS DynamoDB**, the solution processes live streaming events to visualize user interactions with video content and advertisements. 
+
+Through this hands-on guide, you’ll learn how to build, deploy, and extend real-time, event-driven architectures using Snowplow and Kafka, enabling personalized recommendations, real-time insights, and dynamic analytics for streaming platforms. The framework is inspired by common challenges in video streaming, including tracking user behavior, ad engagement, and session activities, with the goal of maintaining up-to-date viewer profiles in DynamoDB.
+
+This accelerator is open source and can serve as the foundation to build practical applications like real-time viewer insights, engagement analytics, ad performance tracking, and personalized recommendations. Whether you're optimizing ad placements or enhancing viewer satisfaction, this guide equips you to unlock the full potential of Snowplow event data.
+
+---
+
+## Solution accelerator code
+[**The code for this infrastructure is available on here on GitHub.**](https://github.com/snowplow-incubator/live-viewer-profiles)
+
+---
+
```

Comment:
would be nice to have a quick intro here of the different pages in this accelerator. Do they have to follow both in sequence?